wagtail/core/tests/test_blocks.py

```diff
@@ -1710,21 +1644,19 @@ class Meta:
                 help_text = "Self-promotion is encouraged"
 
         block = LinkBlock()
-        html = block.render_form(block.to_python({
-            'title': "Wagtail site",
-            'link': 'http://www.wagtail.io',
-        }), prefix='mylink')
 
-        self.assertInHTML('<div class="help"> <svg class="icon icon-help default" aria-hidden="true" focusable="false"><use href="#icon-help"></use></svg>  Self-promotion is encouraged</div>', html)
+        block.set_name('test_structblock')
+        js_args = StructBlockAdapter().js_args(block)
 
-        # check it can be overridden in the block constructor
-        block = LinkBlock(help_text="Self-promotion is discouraged")
```

Comment:
It would be good to include the equivalent of this in the new test - i.e. confirm that when `LinkBlock(help_text="Self-promotion is discouraged")` is passed to `StructBlockAdapter.js_args`, the overridden help text comes through in the result.

Reply:
Added, thanks!